src/PrestaShopBundle/Form/Admin/Sell/Order/CartSummaryType.php

```diff
@@ -79,8 +88,8 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ])
             ->add('order_state', ChoiceType::class, [
                 'choices' => $this->orderStatesChoiceProvider->getChoices(),
-                'required' => false,
-                'placeholder' => false,
+                'required' => true,
+                'placeholder' => $this->translator->trans('Choose a feature', [], 'Admin.Catalog.Feature'),
```

Comment:
@MatShir Need wording

Reply:
```suggestion
                'placeholder' => $this->translator->trans('Choose an order status', [], 'Admin.Orderscustomers.Feature'),
```

This one already exists and seems fitting

Reply:
Corrigé ;)